client/components/tinymce/index.jsx

```diff
@@ -178,6 +181,12 @@ module.exports = React.createClass( {
 
 		this.localize();
 
+		let toolbar1 = 'wpcom_add_media,';
+		if ( config.isEnabled( 'post-editor/contact-from' ) ) {
+			toolbar1 += 'wpcom_add_contact_form,';
```

Comment:
there won't be a need for more feature flags. This code is temporal and will be reverted as soon as this goes into production. I'd rather keep it as it is if you are ok with this.